libs/langchain/langchain/llms/tensorrt.py

```diff
@@ -0,0 +1,528 @@
+from typing import Any, Callable, Dict, List, Optional, Type, Union
```

Comment:
Can we rename the file as triton_tensorrt.py and make relevant changes in the code

Reply:
Thanks @sumitkbh just made the changes. Let me know if that looks good to you now.